services/core-web/src/tests/components/dashboard/noticeOfWorkHomePage/__snapshots__/NoticeOfWorkTable.spec.js.snap

```diff
@@ -1,121 +1,7 @@
-      "Application Status",
-      "Import Date",
-      "",
-    ]
-  }
->
-  <withStore(Table)
-    align="left"
-    columns={
-      Array [
-        Object {
-          "dataIndex": "mineRegion",
-          "filteredValue": "SW,NE",
-          "filters": Array [],
-          "render": [Function],
-          "sortOrder": false,
-          "title": "Region",
-        },
-        Object {
-          "dataIndex": "nowNum",
-          "filterDropdown": [Function],
-          "filterIcon": [Function],
-          "render": [Function],
-          "sortField": "now_number",
-          "sortOrder": false,
-          "sorter": true,
-          "title": "NoW No.",
-        },
-        Object {
-          "dataIndex": "mineName",
-          "render": [Function],
-          "sortOrder": false,
-          "title": "Mine",
-        },
-        Object {
-          "dataIndex": "nowType",
-          "filterDropdown": [Function],
-          "filterIcon": [Function],
-          "render": [Function],
-          "sortField": "notice_of_work_type_description",
-          "sortOrder": false,
-          "sorter": true,
-          "title": "NoW Type",
-        },
-        Object {
-          "dataIndex": "status",
-          "filterDropdown": [Function],
-          "filterIcon": [Function],
-          "render": [Function],
-          "sortField": "now_application_status_description",
-          "sortOrder": false,
-          "sorter": true,
-          "title": "Application Status",
-        },
-        Object {
-          "dataIndex": "date",
-          "render": [Function],
-          "sortField": "received_date",
-          "sortOrder": false,
-          "sorter": true,
-          "title": "Import Date",
-        },
-        Object {
-          "dataIndex": "verify",
-          "render": [Function],
-          "sortOrder": false,
-          "title": "",
-          "width": 150,
-        },
-      ]
-    }
-    dataSource={
-      Array [
-        Object {
-          "date": "Aug 14 2019",
-          "key": "07e801a0-fa33-4c3b-abcc-ac6df628d483",
-          "mineGuid": "6e9d3426-ebf1-413f-9e5f-c3a71ab56797",
-          "mineName": "Hamilton, Herrera and Mccormick",
-          "mineRegion": "N/A",
-          "nowNum": 44,
-          "nowType": "choice",
-          "source": "N/A",
-          "status": "Approved",
-        },
-        Object {
-          "date": "Jul 21 2019",
-          "key": "8e1536da-644c-4961-976b-b1326fa75825",
-          "mineGuid": "60300a07-376c-46f1-a984-88a813f91438",
-          "mineName": "Thompson-Sullivan",
-          "mineRegion": "N/A",
-          "nowNum": 52,
-          "nowType": "technology",
-          "source": "N/A",
-          "status": "Approved",
-        },
-      ]
-    }
-    locale={
-      Object {
-        "emptyText": <NullScreen
-          type="no-results"
-        />,
-      }
-    }
-    onChange={[Function]}
-    pagination={false}
-    rowClassName="fade-in"
-  />
-</TableLoadingWrapper>
+<ContextConsumer>
```

Comment:
It looks like there's a bug causing the snapshot to render as an empty component, probably something undefined that should be either added to the snapshot definition, or checked in code.

Check setupProps in MineNoticeOfWorkTable.spec.js, I'm thinking you should add a definition for location there.

Reply:
This was generated/changed when I ran the updateAll command. Not too sure about what much of this means yet to be honest.